### ISSUE TITLE
Screen reader compatibility main window

### ISSUE THREAD

**Issue by eburggraaf** [BiglySoftware/BiglyBT#1901]
Hi, I just installed biglybt on windows 10.  It's main screan is hopeless.  The nonvisual Desktop Access (NVDA) screen reader reports only a single tab control, an edit box with nothing in it, and 10 or 15 unknown controls.  Only the status bar showing the upload speed, download speed, and DHT status is the slightest bit useful.

It's time to revisit the interface and wire up accessibility.  I have eclypse and will be happy to help if some one can show me how to get started.

Thanks,

Erik
* OS and version Windows 10 
* BiglyBT Version Number 2.5.0.0
* For *nix: Desktop Environment

**Comment by parg** [comment]
Hi,

We use the SWT UI library (from the same people that make the Eclipse IDE) - it is based on mainly native UI widgets. We rely on it for much of the user interface although most table views are drawn by our own code on a Canvas object in order to get the flexibility we required.

I'm afraid I have no idea what would be involved in making things work with NVDA. Setting up Eclipse should be pretty straight forward although someone was struggling to do so recently - see https://github.com/BiglySoftware/BiglyBT/issues/1900.

**Comment by eburggraaf** [comment]
I have eclypse set up already.  I'm a programming student.  Unfortunately my program focussed heavily on C# and I don't have as much java experience as I would like.  I would be willing to bet that most of the standard componants just need a simple settings tweak.  The custom control might be more work, but anything is better than what we have right now.  I will be happy to take a look at it and do as much of the work myself as I can but don't know how to get started getting the source from github and other little chores that need to be done so I can see the code.

**Comment by eburggraaf** [comment]
Re-opening in case I closed this by accident.  Accessibility certainly isn't fixed, so this shouldn't be closed.

**Comment by tapper82** [comment]
Hi any fix for usage with NVDA yet?
